change/@ni-nimble-components-fd008326-f15c-4746-bd5a-61d77d20bffa.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Reminder to link to this new spec from the top level spec.

Reply:
Done.

Reply:
@atmgrifter00 this PR doesn't show any changes to the top level spec. Am I missing something or did this get overlooked?

Reply:
Oops, good call. I'll update that spec as well.